theano/tensor/opt.py

```diff
@@ -1781,7 +1784,7 @@ def local_alloc_unary(node):
             # Is it really necessary to copy over stack trace here?
             # after all, T.alloc and T.cast should preserve the stack trace from x,
             # but perhaps the trace is lost in "v = node.op(x)"?
```

Comment:
`alloc` and `cast` are not keeping the stack trace from `x`.
I think we should change the comment to say that it is not necessary for the moment to add stack traces to `v`, or the output of `T.cast`, but it could be revisited.